### ISSUE TITLE
Add progress bar to result page for votes

### ISSUE THREAD

**Issue by T4rikA** [jeriox/myHPI#21]

**Comment by jeriox** [comment]
i think there is no preference

**Comment by dasGoogle** [comment]
@BenBals Is this still practical for the alternative voting schemes you implemented and we used?

**Comment by BenBals** [comment]
Yep, there are cool ways to illustrate or animate instant run-off elections. The last tool we used had a slider for rounds and then displayed a bar chart of current vote totals for each round.
